pyproject.toml

```diff
@@ -28,7 +28,7 @@ pylibmc = "^1.6.1"
 psycopg2-binary = "^2.9.9"
 boto = "^2.49.0"
 django-cors-headers = "^4.3.1"
-protobuf = "^4.25.3"
+protobuf = "^5.26.0"
```

Comment:
@DonnieBLT do you have an idea why this listed in the direct dependencies?

Reply:
I believe it would have been in the poetry.lock then? 

Anyway, I don't think we use `protobuf` anywhere. Did I miss it?